pkgs/by-name/sc/sccmhunter/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "sccmhunter";
+  version = "1.0.6-unstable-2024-10-30";
+  pythonRelaxDeps = true;
```

Comment:
This package does not have strict dependency version restrictions

```suggestion
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook